test/uk/gov/hmrc/cataloguefrontend/JsonData.scala

```diff
@@ -22,78 +22,21 @@ import uk.gov.hmrc.cataloguefrontend.shuttering.{ShutterStatusValue, ShutterType
 import java.time.Instant
 
 /**
-  * Created by armin.
-  */
+ * Created by armin.
+ *
+ * TODO: Refactor the data here into files under the
```

Comment:
Can these TODO comments be removed now?
You can also remove the "Created by" comment while we're here.

Reply:
Or just keep the TODOs locally

Reply:
The point for the TODOs was because they need to be TOdone :D And it might not always be me TODO'ing them so a local copy wouldn't really help much.  I've added https://jira.tools.tax.service.gov.uk/browse/BDOG-2593 so that the work to tidy up isn't lost

Reply:
Can they be removed now we have a ticket?

Reply:
I guess the point they can be removed is when they are TOdone :)